test/integration/scripting_spec.mjs

```diff
@@ -1993,18 +1991,9 @@ describe("Interaction", () => {
 
           await page.click(getSelector("26R"));
           await page.type(getSelector("26R"), "abcde", { delay: 10 });
-
           await page.click(getSelector("23R"));
-          // eslint-disable-next-line no-restricted-syntax
-          await waitForTimeout(10);
-          await page.click(getSelector("26R"));
-
-          await kbSelectAll(page);
-          await page.keyboard.press("Backspace");
-
+          await clearInput(page, getSelector("26R"));
```

Comment:
The previous code is changed to `clearInput` here because since commit 3f51da147cb93b34f8769dfe3e463cfae397bfd2 the helper function takes care of doing all required operations (making the code here simpler) and waiting until the input is actually fully cleared (which didn't happen here before).